scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -579,6 +579,11 @@ def load_from_file(self, file, filter_platform=[]):
                     self.options.fixture
                 )
 
+                for h in self.hwm.duts:
+                    if h.platform == platform.name:
+                        if instance.testsuite.harness_config.get('fixture') in h.fixtures:
+                            instance.run = True
```

Comment:
if true, we can break out right? no need to loop further

Reply:
@hakehuang ping